packages/kg-default-nodes/lib/nodes/signup/SignupParser.js

```diff
@@ -50,21 +50,21 @@ export class SignupParser {
                             const disclaimer = domNode.querySelector('p')?.textContent || '';
                             const backgroundImageSrc = layout === 'split'
                                 ? domNode.querySelector('.kg-signup-card-image')?.getAttribute('src')
-                                : domNode.querySelector('.kg-signup-card-container')?.getAttribute('style')?.match(/url\((.+)\)/)?.[1] || '';
-                            const backgroundColor = domNode.querySelector('.kg-signup-card-container')?.style.backgroundColor || '';
+                                : domNode.querySelector('.kg-signup-card-text')?.getAttribute('style')?.match(/url\((.+)\)/)?.[1] || '';
+                            const backgroundColor = domNode.querySelector('.kg-signup-card-text')?.style.backgroundColor || '';
                             const buttonColor = domNode.querySelector('.kg-signup-card-button')?.style.backgroundColor || '';
                             const buttonText = domNode.querySelector('.kg-signup-card-button-default')?.textContent?.trim() || 'Subscribe';
                             const buttonTextColor = domNode.querySelector('.kg-signup-card-button')?.style.color || '';
                             const textColor = domNode.querySelector('.kg-signup-card-success')?.style.color || '';
-                            const alignment = domNode.querySelector('.kg-signup-card-container')?.classList.contains('align-center') ? 'center' : 'left';
+                            const alignment = domNode.querySelector('.kg-signup-card-text')?.classList.contains('align-center') ? 'center' : 'left';
```

Comment:
Note to self: lets rename this to kg-align-center or something similar

Reply:
✅